src/js/flows/initConnection.ts

```diff
@@ -4,13 +4,14 @@ import {Cluster} from "../state/Clusters/types"
 import refreshSpaceNames from "./refreshSpaceNames"
 import {globalDispatch} from "../state/GlobalContext"
 import ConnectionStatuses from "../state/ConnectionStatuses"
+import brim from "../brim"
 
-export const initConnection = (cluster: Cluster) => (
+export const initConnection = (cluster: Cluster, isNew = false) => (
```

Comment:
Instead of an additional conditional param, could we check the state for this cluster's id? Or check for this id in the ConnectionStatuses?